src/headless/converse-chatboxes.js

```diff
@@ -7,6 +7,7 @@ import "./converse-emoji";
 import { Collection } from "@converse/skeletor/src/collection";
 import { _converse, api, converse } from "./converse-core";
 import log from "./log";
+import Favico from 'favico.js-slevomat';
```

Comment:
Why are you using the forked version of favico.js ?

Reply:
latest version 0.3.11 from this commit in 2018
https://github.com/slevomat/favico.js/commit/c0f5ea878e7a4d1ae2b18cdc6d30a093e7961d40

The original project is at version 0.3.10 and has not be touched since 2016